fiftyone/server/routes/get_similar_labels_frames.py

```diff
@@ -0,0 +1,325 @@
+                                                    ]
+                                                },
+                                                "then": "$doc.v.detections",
+                                            },
+                                            {
+                                                "case": {
+                                                    "$eq": [
+                                                        "$doc.v._cls",
+                                                        "Polylines",
+                                                    ]
+                                                },
+                                                "then": "$doc.v.polylines",
+                                            },
+                                            {
+                                                "case": {
+                                                    "$eq": [
+                                                        "$doc.v._cls",
+                                                        "Keypoints",
+                                                    ]
+                                                },
+                                                "then": "$doc.v.keypoints",
+                                            },
+                                        ],
+                                        "default": [],
+                                    }
+                                },
+                                "frameNumber": "$frame_number",
+                            }
+                        },
+                        # From the nested items, filter and transform:
+                        # - Use $filter to select items that are spatial labels
+                        #   (as defined in SPATIAL_LABEL_TYPES) and whose
+                        #   instance id matches the given instance.
+                        # - Use $map to convert each matching label into its
+                        #   _id.
+                        # The resulting document contains:
+                        # - labelIds: an array of label _ids.
+                        # - frameNumber: the frame number.
+                        {
+                            "$project": {
+                                "labelIds": {
+                                    "$map": {
+                                        "input": {
+                                            "$filter": {
+                                                "input": "$nestedItems",
+                                                "as": "item",
+                                                "cond": {
+                                                    "$and": [
+                                                        {
+                                                            "$in": [
+                                                                "$$item._cls",
```

Comment:
`_cls` is not guaranteed. Can we use the instead?

Reply:
done